### PR TITLE
Added Commercial Section to Geotagging wiki.

### PR DESCRIPTION
**Pull request by ptsneves**
Airborne Projects released a geotagging
solution which might be useful for some people.
The addition is a brief description of the
board, how it connects to the APM and 3 images.

The spirit of the section is similar to the
one found on http://ardupilot.org/copter/docs/common-frsky-telemetry.html#hardware-solutions
which encouraged people to add their own alternative solution.

### CODE REVIEW

File: common/source/docs/common-geotagging-images-with-mission-planner.rst [hunk 1]
```diff
@@ -4,312 +4,337 @@
...
+=========================================
 
-Additional information
-======================
+DROTAG - Airborne Projects
+--------------------------
 
-This section contains links to additional information/information for
-working with specific cameras:
+The `DROTAG <https://www.airborneprojects.com/product/drotag-onboard-image-tagging-computer/>`_ (airborneprojects.com) is a small board (55x27mm at 21g) that connects to an APM compatible Flight Controller (like Pixhawk) through the one of the any available TELEMETRY port and through a MicroUSB cable to the camera. That's it. Nothing more is needed.
```
Comment (hamishwillee) [inline common/source/docs/common-geotagging-images-with-mission-planner.rst]: This says that it is a small board, and that nothing more is needed ... but it doesn't say "needed for what"? Best to start by saying DROTAG is a board that you can add which will automatically geotag files when this is specified in mission planner (or similar)

File: common/source/docs/common-geotagging-images-with-mission-planner.rst [hunk 2]
```diff
@@ -4,312 +4,337 @@
...
-working with specific cameras:
+The `DROTAG <https://www.airborneprojects.com/product/drotag-onboard-image-tagging-computer/>`_ (airborneprojects.com) is a small board (55x27mm at 21g) that connects to an APM compatible Flight Controller (like Pixhawk) through the one of the any available TELEMETRY port and through a MicroUSB cable to the camera. That's it. Nothing more is needed. 
 
--  `Geotag GoPro Images with a Pixhawk Log File <http://tuffwing.com/support/geotag_gopro_images_with_pixhawk_log.html>`__
-   (tuffwing.com)
+With the appropriate mission planning you will have your taken pictures automatically geotagged by DROTAG in the MicroSD card at the time you land, with no need for any post-processing. DROTAG also has a way to configure most of your camera settings. This is specially important if you use your aerial camera for other earthly uses.
 
-Acknowledgements
-================
+Airborne Project's also offers a comprehensive `DROTAG manual <https://www.airborneprojects.com/wp-content/uploads/2016/06/Manual.Gstreamer.Arietta.pdf>`_ (airborneprojects.com) with before flight checklists detailed explanations for diagnosis and other tips.
```
Comment (hamishwillee) [inline common/source/docs/common-geotagging-images-with-mission-planner.rst]: "offers", might be better as "provides"

> with before flight checklists detailed explanations for diagnosis and other tips.

The above needs punctuation.

File: common/source/docs/common-geotagging-images-with-mission-planner.rst [hunk 3]
```diff
@@ -4,312 +4,337 @@
...
 
-This tutorial was created to show you how does it work. Original work
-from Sandro Benigno and Guto Santaella updated by Jesus Alvarez.
+
+ .. image:: https://www.airborneprojects.com/wp-content/uploads/2016/06/text_image_1-1200x548.jpg
```
Comment (hamishwillee) [inline common/source/docs/common-geotagging-images-with-mission-planner.rst]: Images look indented. Should be flush to left.

**Comment by hamishwillee** [comment]
Modulo the comments above this sounds sane. @rmackay9 @magicrub - you OK with this addition? (the change is the section at the end, most easily viewed using the rich diff: https://github.com/ArduPilot/ardupilot_wiki/pull/481/files?short_path=da4a855#diff-da4a85519ab7fa6627c01dcc2edf9d23

**Comment by magicrub** [comment]
whyyyyyyyy must every line change! grr. Hey @github please add a "ignore whitespace" option in the diffs!!!

**Comment by magicrub** [comment]
Check this out! I sent in a complain to github and a human (I think) responded in about an hour:

> > ------ My Complaint -----
> > Please add a feature to allow ignoring whitespace in a diff. I often get pull-requests on our wiki that has the entire page changed but really it's just one line and a ton of whitespace. I want to know the actual content change and don't care about the CRLF changes.
>
> ------ GitHub's Response -----
> Hi Tom,
> 
> Thanks for your feedback.
> 
> I'm not sure if you are aware, but we have a special URL parameter that you can add to ignore whitespace in a diff. You add ?w=1 to the end of the URL and whitespace will be ignored.
> 
> I'll still add your feature request for better support for ignoring whitespace to out internal feature request list, because I think we could be a little clearer on how to ignore whitespace.
> 
> I hope this helps. Let us know if you have any other questions.
> 
> Thanks,
> Alex
>

**Comment by hamishwillee** [comment]
@magicrub That is awesome - https://github.com/ArduPilot/ardupilot_wiki/pull/481/files?w=1 :-) They definitely need a button for that, but so much better. 

This change now looks "good enough". I may do further minor tidy in future. You OK for me to merge (or please do!)

**Comment by magicrub** [comment]
Merged

**Comment by ptsneves** [comment]
I just found out that `git add -up` does not have a ignore white space option so the only way to produce patches without white space changes is through saving the diff and applying it. 

If you need more changes let me know. I also put instructions on the [post](
https://community.emlid.com/t/drotag-beta-testing-trigger-geo-reference-and-configure-your-camera/3179/1) with instructions to run DROTAG free of charge for a month, so if anybody is interested feel free.

**Comment by magicrub** [comment]
The white space option is in your global config.

**Comment by CraigElder** [comment]
It seems appropriate that Tom is putting the magicrub on whitespace

**Comment by ptsneves** [comment]
The left alignment which was requested resulted in the page looking defective, as can be seen in the live version.

It is strange because in this [live editor](http://rst.ninjs.org/?n=855781498e8fcb2c12b4f0ce12127a6e&theme=basic), the page is rendered correctly.

This time I installed sphinx and built a local copy of the documentation and removed the align directive, resulting in a correctly formatted page. Please correct me if I am wrong.

I also am not sure how github works regarding the modification made in a new commit to reflect the removal of the align directive.

https://github.com/ptsneves/ardupilot_wiki/commit/3c7c49533fa6b8ab2c3fe8df76ad72c6ef4eda30

Best regards
Paulo Neves

**Comment by hamishwillee** [comment]
@ptsneves Thanks for that. I fixed it in #493 rather than using your commit because I wasn't sure how to merge that either.

Your fix was correct. When I suggested align left I meant removing the additional indentation on the left, not actual alignment of the images. Apologies for not be more clear. On this wiki almost all images are "none" aligned in source, resulting in left alignment on individual lines.

Thanks!

**Pull request merged**